docs/03-tutorials/service-management/smgt-01-hb-broker-registration.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Register a broker in the Service Catalog
+---
+
+This tutorial shows you how to register a broker in the Service Catalog. The broker can be either a Namespace-scoped ServiceBroker or a cluster-wide ClusterServiceBroker. Follow this guide to register a cluster-wide or Namespace-scoped version of the sample [UPS Broker](https://github.com/kubernetes-incubator/service-catalog/tree/master/charts/ups-broker).
+
+## Prerequisites
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+* [helm](https://github.com/helm/helm#install)
+
+## Steps
+
+1. Clone the [`service-catalog`](https://github.com/kubernetes-incubator/service-catalog) repository:
+    ```
+    git clone https://github.com/kubernetes-incubator/service-catalog.git
+    ```
+
+2. Check out one of the official tags. For example:
+
+    ```
+    git fetch --all --tags --prune
+    git checkout tags/v0.1.39 -b v0.1.39
+    ```
+3. Create the `ups-broker` Namespace:
+    ```
+    kubectl create namespace ups-broker
+    ```
+
+4. Run this command to install the chart with the `ups-broker` name in the `ups-broker` Namespace:
+      ```
+     helm install ups-broker ./charts/ups-broker --namespace ups-broker
+     ```
+
+5. Register a broker:
+  * Run this command to register a ClusterServiceBroker:
```

Comment:
I'd suggest to have the same sentence structure for both options - either "To..., run..." (preferred, because it mentions the purpose first) or "Run... to...".